oaknational/Oak-Web-Application
# feat: resolve css to rem units

## Description

- update `parse` functions of css helpers, so that they resolve to 'rem' units
- there are a few edge cases that haven't changed, but this should mostly cover it



## How to test

1. Go to https://deploy-preview-1764--oak-web-application.netlify.thenational.academy
2. Style should appear unchanged
3. Change your browser setting so that base font size is smaller/larger
4. You should see this change reflected in the style on the site

## Screenshots

E.g.

<img width="1440" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/12934669/eb1c5dda-89dc-40c1-a3de-c8eea56a71f4">

<img width="1440" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/12934669/6d2b6308-3b68-4762-a073-3082e755d9a6">



## Checklist
